client/src/main/java/io/paradoxical/cassieq/api/client/CassandraQueueApi.java

```diff
@@ -27,26 +33,22 @@ static CassandraQueueApi createClient(String baseUri) {
         return service;
     }
 
-    @POST("api/v1/queues")
-    Call<ResponseBody> createQueue(@Body QueueCreateOptions queueName);
+    @POST("api/v1/queues") Call<ResponseBody> createQueue(@Body QueueCreateOptions queueName);
 
-    @GET("api/v1/queues/{queueName}/messages/next")
-    Call<GetMessageResponse> getMessage(@Path("queueName") QueueName queueName);
+    @GET("api/v1/queues/{queueName}/messages/next") Call<GetMessageResponse> getMessage(@Path("queueName") QueueName queueName);
 
-    @GET("api/v1/queues/{queueName}/messages/next")
-    Call<GetMessageResponse> getMessage(@Path("queueName") QueueName queueName, @Query("invisibilityTime") Long invisibilityTimeSeconds);
+    @GET("api/v1/queues/{queueName}/messages/next") Call<GetMessageResponse> getMessage(
+            @Path("queueName") QueueName queueName,
+            @Query("invisibilityTime") Long invisibilityTimeSeconds);
 
-    @POST("api/v1/queues/{queueName}/messages")
-    Call<ResponseBody> addMessage(@Path("queueName") QueueName queueName, @Body String message);
+    @POST("api/v1/queues/{queueName}/messages") Call<ResponseBody> addMessage(@Path("queueName") QueueName queueName, @Body Object message);
 
-    @POST("api/v1/queues/{queueName}/messages")
-    Call<ResponseBody> addMessage(
+    @POST("api/v1/queues/{queueName}/messages") Call<ResponseBody> addMessage(
```

Comment:
not a huge fan of annotations on one line. we should consider updating the intellij styles.